Files/Interacts/Interaction.cs

```diff
@@ -1081,12 +1082,47 @@ public async Task PasteItems(DataPackageView packageView, string destinationPath
             {
                 foreach (IStorageItem item in itemsToPaste)
                 {
+                    if (!copyDuplicate && !string.IsNullOrEmpty(item.Path) &&
```

Comment:
Remove this block, as long as this dialog is not used.

Reply:
I created that block to address #1546, but since this isn't the scope of the pr I will remove it for now and address in a next pr.